src/planscape/planning/views.py

```diff
@@ -373,21 +397,23 @@ def create_scenario(request: HttpRequest) -> HttpResponse:
         # Create a default scenario result.
         # Note that if this fails, we will have still written the Scenario without
         # a corresponding ScenarioResult.
-        scenario_result = ScenarioResult.objects.create(
-            scenario=scenario)
+        scenario_result = ScenarioResult.objects.create(scenario=scenario)
         scenario_result.save()
+        # async popen call
+        call_forsys(scenario.pk)
```

Comment:
In this flow, which flow is responsible for updating the status of the scenarioresult (e.g. pending -> running -> success|failure) ?

Reply:
Ultimately, IMO it's SIG's call as to how much logic to put in the R code vs the python code that is invoking it.  While the EP was originally written so that it handles most everything except calling forsys (which must be done in R), moving more logic into the R code is also ok.  That said, I recommend writing out a plan for this so that all parties are clear on which part of the code is responsible for what.  I'm under the impression that the R code isn't called without a python wrapper.